package.json

```diff
@@ -55,12 +60,12 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^3.6.2",
     "remark-cli": "^12.0.1",
-    "stylelint": "^16.23.0"
+    "stylelint": "github:stylelint/stylelint#v17"
   },
   "peerDependencies": {
-    "stylelint": "^16.23.0"
+    "stylelint": "^17.0.0"
   },
   "engines": {
-    "node": ">=18.12.0"
+    "node": ">=20.19.0 || >=22.13.0"
```

Comment:
question

`|| >=22.13.0` is already covered by >=20.19.0, right?

v23.5.0, v22.13.0, v20.19.0 | This feature no longer emits an experimental warning by default, though the warning can still be emitted by --trace-require-module.
-- | --

Based on that I suppose that you want ranges instead.
i.e. exclude 22.0.0–<22.13.0 and 23.0.0–<23.5.0

related:  stylelint/stylelint#8867